frontend/static/projectDetails.html

```diff
@@ -698,12 +698,10 @@ <h4>Field : {{fieldForm.name}}</h4>
                     </div>
 
                     <div style="margin-top: 20px;width: 100%;display: flex">
-                    
+                    <div title="Enabled for Number Type Fields">
                         <paper-checkbox on-change="displayField" id="enableScoringCoefficientEdit"
-                                        checked="{{fieldForm.enable_scoring_coefficient}}" style="margin-top:30px;margin-right: 20px">Ranked
-                            Search</paper-checkbox>
-                        <paper-tooltip for="enable_scoring_coefficient">Enable ranked Search</paper-tooltip>
-                        <paper-input style="float:right;width:50%;visibility: {{fieldForm.visibility}}" class="projectInputStyle" label="Scoring Coefficient:"  value="{{fieldForm.scoring_coefficient}}" id="rankingm" auto-validate pattern="([0-9]*[.])?[0-9]+" error-message="Numbers or Floats only!"></paper-input>
+                                        checked="{{fieldForm.enable_scoring_coefficient}}" disabled="{{editType}}" style="margin-top:30px;margin-right: 20px">Use in Document Search</paper-checkbox></div>
```

Comment:
This should be changed to `Use in document scoring`.   Is there any way to not have these tags duplicated in the html?

Reply:
Sometimes I get `Use in Document Search`, sometimes I get `Use in Document Scoring`.  I'm not sure why either time.